### PR TITLE
Destroyed trees

### PR DESCRIPTION
**Pull request by DnAp**
Now you can destroy trees

### CODE REVIEW

File: mods/ra2/rules/defaults.yaml [hunk 1]
```diff
@@ -550,6 +551,12 @@
 	RenderSprites:
 		Palette: terrain
 	AutoSelectionSize:
+	Targetable:
+		TargetTypes: Ground, Wall, C4, DetonateAttack
+	Health:
+		HP: 200
+	Armor:
+		Type: Concrete
```
Comment (GraionDilach) [inline mods/ra2/rules/defaults.yaml]: Trees had Armor: wood in TS and RA2.

**Comment by DnAp** [comment]
Fix it

**Comment by Iamgoofball** [comment]
https://github.com/OpenRA/ra2/pull/1

**Comment by abcdefg30** [comment]
This was merged: https://github.com/OpenRA/ra2/pull/2

**Pull request closed without merging**